umap/static/umap/js/umap.features.js

```diff
@@ -45,6 +45,15 @@ L.U.FeatureMixin = {
         return this.datalayer && this.datalayer.isRemoteLayer();
     },
 
+    getSlug: function () {
+        return this.properties[this.map.options.slugKey || 'name'] || '';
```

Comment:
slug ?
![image](https://user-images.githubusercontent.com/6597721/77249952-f871b980-6c44-11ea-9878-8628b593f159.png)

:smile:

Reply:
You mean `slug` instead of `name` ?

Reply:
Ah, ok `slug` means `limace`, I didn't know ;)